### DISCUSSION TITLE
Request: View filename instead of id3 tag data

### DISCUSSION THREAD

**Discussion by clintonthegeek** [PaulWoitaschek/Voice#1425] [Ideas]
Hi! Love this app, use it every day!

I've many .mp3 files with unhelpful name-tags in their metadata. The person releasing the files just put the same info in every field for ever file. But the filenames are useful.

Right now I have no way of knowing what file I'm listening to or opening in Voice. I either have to go and re-tag all the files or try and guess by the length.

Can you add an option to display all files by filename instead of metadata?

**Comment by nimbus77** [comment]
This would be useful since some producers of audiobooks that I've bought don't know how to tag properly. But it would have to be a per book setting.

**Comment by Keavon** [comment]
This is extremely important. It is absolutely the biggest flaw with the app right now, and the sole reason I have been trying to switch to something else (however, annoyingly, nobody else has made anything better). The app is in dire need of showing file names or else navigation is entirely useless for nearly every book.

**Comment by thelele** [comment]
I agree with you guys. We need an option to display audiobook name by Folder name, and parts of audiobook (like if you have 5 chapter in mp3 for one book) by name too !

I use to love so much this app when the name was different, it uses to use the file name and was awesome, unfortunately the app change several time the way of which name to use, I've spent several day trying to retag correctly hundred of books but it's always a mess, and some don't get tag correctly etc...

In short being able to chose display by file name for the audiobook name and chapter would be so awesome. Like other I love this app well made except for the filename management.


By the ways, is there a way to access old apk ?

**Comment by bjohas** [comment]
It would be great if this was possible, e.g. book name from folder name. At the moment I'm also messing around with the tags in my book library to get a reasonable listing.  I'd prefer it to be a setting in the preferences, as I'd just like to use this for all books.

Otherwise the app is really great!
